Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +36,63 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message))
+            {
+                return "";
+            }
+
+            // split the message
+            string[] stringArray = message.Split(Environment.NewLine);
+            int maxLength = -1;
+
+
+            // calculate max length
+            foreach (string item in stringArray)
+            {
+                if (maxLength < item.Length)
+                {
+                    maxLength = item.Length;
+                }
+            }
+
+            // calculate table borders
+            char corner = '+';
+            int width = maxLength + padding * 2;
+            string topLine = new string('-', width);
+            string bottomLine = topLine;
+            string horizontalPadding = padding > 0 ? new string(' ', padding) : "";
+            string leftBorder = "|" + horizontalPadding;
+            string rightBorder = horizontalPadding + "|";
+            string verticalPadding = "";
+            string newLine = Environment.NewLine;
+
+            if (padding > 0)
+            {
+                for (int i = 0; i < padding; i++)
+                {
+                    verticalPadding += leftBorder + new string(' ', maxLength)
+                        + rightBorder + newLine;
+                }
+            }
+
+            // top border
+            string result = $"{corner}{topLine}{corner}{newLine}";
+            result += verticalPadding;
+
+            //rows
+            foreach (string item in stringArray)
+            {
+                string sentence = item.PadRight(maxLength);
+                result += $"{leftBorder}{sentence}{rightBorder}{newLine}";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a StringBuilder instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069037)